pandas/plotting/_matplotlib/misc.py

```diff
@@ -144,7 +144,9 @@ def normalize(series):
     df = frame.drop(class_column, axis=1).apply(normalize)
 
     if ax is None:
-        ax = plt.gca(xlim=[-1, 1], ylim=[-1, 1])
+        ax = plt.gca()
+        ax.set_xlim([-1, 1])
+        ax.set_ylim([-1, 1])
```

Comment:
I guess you can even use ``ax.set_xlim(-1, 1)``, without extra brackets.
The same for other cases as well.

Reply:
Done